services/app-api/form/submitAny.js

```diff
@@ -90,8 +93,9 @@ export const submitAny = async (event, config) => {
 
     const results = await Promise.all(promises);
     console.log("ID Matching results are: ", results);
-    if (results[0] != undefined)
-      results.forEach((errorMsg) => {
+    results
+      .filter((errorMsg) => errorMsg != undefined)
+      .forEach((errorMsg) => {
```

Comment:
I was seeing an odd bug whereby the first result was undefined and second result was an error in the array, therefore updated this to filter out all undefined and process rest rather than just look at first item in array